### PR TITLE
Mybet command

### LINKED ISSUE #15: As an user I want to see what bet I have placed using discord command
Additional info:

### LINKED ISSUE #20: Deno Action performs Deno fmt check on dist folder with bundled javascript
Additional info:
The dist directory is serving the frontend and has bundled javascript. Deno action will always fail because of this file and make the action therefore obsolete. We should investigate and find a solution to exclude dist from the action

### PR DESCRIPTION
**Pull request by xTomsko**
closes #15

### CODE REVIEW

**Comment by alexsc23** [comment]
Add one of these options: https://deno.land/manual/tools/formatter#ignoring-code for successful checks.

**Comment by xTomsko** [comment]
> Add one of these options: https://deno.land/manual/tools/formatter#ignoring-code for successful checks.

@alexsc23 Well it is nothing wrong with the changed files. The file that causes the crash is not manually created, it is the bundled JavaScript. For reference #20

**Comment by alexsc23** [comment]
Fixed problem with deno fmt & lint.
But now fails at tests while connecting to mongodb.

**Review by alexsc23 — APPROVED** [review]

File: web/backend/db/mongo.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ interface UserSchema {
   phone: string;
 }
 
-interface BetsSchema {
+export interface BetsSchema {
```
Comment (Lesoge) [inline web/backend/db/mongo.ts]: prob extra file for mongo schemas?

Comment (xTomsko) [reply web/backend/db/mongo.ts]: sounds like a plan

**Review by TheMuppet — APPROVED** [review]

**Pull request merged**